### ISSUE TITLE
consider improving accessbility/usability of report link

### ISSUE THREAD

**Issue by danmarsden** [mattan-d/moodle-plagiarism_originality#16]
The report link is provided via a pdf icon that has the alt text "PDF" and it could be easily missed by a teacher unfamiliar with originality - ideally the alt text on the link should be something like "View originality report" rather than "PDF" and you might even consider making the "100%" score text clickable so that it loads the report too.

not a blocker for approval - just might be worth improving slightly.

**Comment by mattan-d** [comment]
Enhance report link accessibility and usability
